src/ORM/Table.php

```diff
@@ -1226,6 +1208,35 @@ public function get($primaryKey, $options = [])
     }
 
     /**
+     * @param int|string|array $primaryKey Primary key value.
+     *
+     * @return array Conditions.
+     */
+    public function primaryKeyToConditions($primaryKey)
```

Comment:
I had it protected first, but @thinkingmedia mentioned it could be useful from the outside.
I am fine reverting this, though.